Documentation/Configuration/Reference/TxSolrSearch.rst

```diff
@@ -1425,6 +1425,8 @@ variants.limit
 
 Limit of expanded documents.
 
+Though this setting limits the returned variants, you still can get the number of existing variants, it's set in "document.variantsNumFound" (since EXT:solr 9)
```

Comment:
;) since EXT:solr 9

Reply:
Right, should be EXT:solr 10

Fixed it